blackbox/docs/sql/ddl.txt

```diff
@@ -662,7 +662,7 @@ Here is a simple Example::
     ...     kstem
     ...   ),
     ...   CHAR_FILTERS (
-    ...     mapping
+    ...     html_strip
     ...   )
     ... );
     CREATE OK, 1 row affected (... sec)
```

Comment:
pls add an at least hidden statement which is using the custom analyzer to verify that it really works